testing/Cargo.toml

```diff
@@ -15,7 +15,7 @@ serde_json = { version = "1.0", optional = true }
 
 [dev-dependencies]
 criterion = "0.3"
-trybuild = "1.0"
+trybuild = "1.0.55"
```

Comment:
This is not pinning it. If you want to actually pin it it should be prefixed with =.

Reply:
Does "1.0.55" mean "1.0.x" with "x >= 55" then? I guess this also OK, but it shouldn't be some ancient version.

Reply:
It actually means 1.x with x > 0.55.